### ISSUE TITLE
Auxiliary loss

### ISSUE THREAD

**Issue by lmz-whu** [Vladkryvoruchko/PSPNet-Keras-tensorflow#51]
Hello, thanks for your works! Have you implemented the auxiliary loss? And where is it?
